crypto/pem/pem_lib.c

```diff
@@ -729,7 +729,7 @@ static int get_name(BIO *bp, char **name, unsigned int flags)
 {
     char *linebuf;
     int ret = 0;
-    size_t len;
+    long int len;
```

Comment:
Why a `long`? The return type of `BIO_gets()` is an int. `len` is also used as a parameter to `sanitize_line` and `pem_malloc` both of which also expect an `int`. So `int` seems more appropriate to me - avoids unnecessary implicit casts.